build.gradle

```diff
@@ -54,7 +54,7 @@ task updateRuntimeLibraries(type: UpdateLibrariesTask) {
 task updateShadeLibraries(type: UpdateLibrariesTask) {
 	directory = librariesShadeDirectory
 	libraries = [
-		[name: 'raknetserver-8.jar', url: 'https://build.true-games.org/job/RakNetServer/29/artifact/target/RakNetServer.jar'],
```

Comment:
Are you testing if I'd see it? Trying to sneak this in buddy? 😉

Reply:
Probably he included his own RakNetServer fork for testing and forgot to remove 😛 

Or maybe it is a bitcoin miner, who knows

Reply:
@7kasper  sorry i iets yesdog branch of raknet but i fargot to remove that because i use it am so sorry

Reply:
And am not a bitcoin miner they dont profit any more of them

Reply:
I know what it is haha, but we should get that brange merged in the 'official' raknet first.

Reply:
ah damn forgot that was in yasunas pr, will switch back